crypto/CMakeLists.txt

```diff
@@ -16,11 +18,31 @@ FetchContent_Declare(
 set(gtest_force_shared_crt ON CACHE BOOL "" FORCE)
 FetchContent_MakeAvailable(googletest)
 
+if(NOT "${CMAKE_GENERATOR}" MATCHES "(Win64|IA64)")
```

Comment:
Please add a comment here explaining the logic.

Reply:
I've added comments and used a better method in the newest commit.